src/styles/modules/_m-tooltip.sass

```diff
@@ -8,17 +8,17 @@
     margin-left: auto
     margin-right: 0
     right: 0
-    background: $color-white
+    background: $color_white
     border-radius: $radius_l
-    box-shadow: $shadow-callout
+    box-shadow: $shadow_callout
     position: relative
     display: inline-block
 
     &::after
       +border(8px solid color(ablack))
```

Comment:
this should be a rem-fallback and we should use a color token

Reply:
How will the `rem-fallback` function work with the `border` function? I can't find where either of these mixins are defined. I'd love to do away with the rem-fallback function by adding it to the build process as we discussed with Kevin.

Reply:
@sgashler it would look like this this `@include rem-fallback("border", 8px solid color(ablack))`

We wouldn't use the border function.  Instead, we would use the rem-fallback mixin.  I would also love to get rid of rem-fallback.  If you can create a proposed process for this and a ticket, I would love to try and get that moving forward!